### DISCUSSION TITLE
用户输入信息缺失时，function_call返回参数问题

### DISCUSSION THREAD

**Discussion by michaelxu1107** [zai-org/ChatGLM3#487] [Q&A]
当用户输入信息缺失时，如何让function_call返回的arguments中的参数value值设置为未知或者其他自定义值，以便后续函数接口特殊处理？谢谢

**Comment by zRzRzRzRzRzRzR** [comment]
#335 中已经提到无法处理缺少参数的情况，你可以尝试在Function Call之前加入参数数量判定（手动写规则），来避免这个问题，目前这类问题无法避免
